pkg/sfu/downtrack.go

```diff
@@ -1217,22 +1200,17 @@ func (d *DownTrack) writeBlankFrameRTP(duration float32, generation uint32) chan
 	return done
 }
 
-func (d *DownTrack) writeOpusBlankFrame(hdr *rtp.Header, frameEndNeeded bool) (int, error) {
+func (d *DownTrack) getOpusBlankFrame(_frameEndNeeded bool) ([]byte, error) {
```

Comment:
Simplifying these. Actually these do not need to be struct method (expect for the VP8 one), but because of the VP8 one leaving these here.